pkg/install/install.go

```diff
@@ -22,6 +22,7 @@ func CmdInstall() *cobra.Command {
 		Short: "Install the operator and create the noobaa system",
 		Run:   RunInstall,
 	}
+	cmd.Flags().Bool("use_graceful_finalizer", true, "Create NooBaa system witout graceful finalizer")
```

Comment:
No 100% sure, but setting the default value as true can be misleading.
As far as I understand using flags (like use_graceful_finalizer) will set the value to true and in order to reverse it one needs to use `-use_graceful_finalizer=flase`
I think it is more straight forward to use a default bool flag as false.
also, the flag and the description are confusing to me as the flag is "use" and the description is "without"

What do you think?

Reply:
Changed the flag's name to skip_graceful_flag and reversed the default value to false
 :)